src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -15,6 +15,10 @@
      */
     public function confirmToProceed($warning = 'Application In Production!', $callback = null)
     {
+        if (! $this->input->isInteractive()) {
+            return true;
```

Comment:
I'm with @GrahamCampbell. imo when running non-interactively the shell should:
- Fill in default values for all actions that have them unless specified by a command line option (if it is even supported)
- Stop execution in situations that require user input (e.g. confirmation) unless given explicit instructions to continue with a given input like when using the `-y` (yes), `-n` (no), `--force` flags to specify the behavior.

This is the least surprising behavior and (at least from my experience) matches what shell commands typically do.